<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FusiondlsGUI</class>
 <widget class="QMainWindow" name="FusiondlsGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1691</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FusiondlsGUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QWidget" name="field_plot" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="gamma_sheath_label">
            <property name="text">
             <string>gamma_sheath</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="gamma_sheath_input">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Tt_label">
            <property name="text">
             <string>Tt</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="Tt_input">
            <property name="text">
             <string>0.5</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="qpllu0_label">
            <property name="text">
             <string>qpllu0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="qpllu0_input">
            <property name="text">
             <string>1e9</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="nu_label">
            <property name="text">
             <string>nu</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="nu_input">
            <property name="text">
             <string>1e20</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="nu0_label">
            <property name="text">
             <string>nu0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="nu0_input">
            <property name="text">
             <string>1e20</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="cz0_label">
            <property name="text">
             <string>cz0</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="cz0_input">
            <property name="text">
             <string>0.02</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="alpha_label">
            <property name="text">
             <string>alpha</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="alpha_input">
            <property name="text">
             <string>1000</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="controlVariableLabel">
            <property name="text">
             <string>control variable</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="controlVariableComboBox">
            <item>
             <property name="text">
              <string>density</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>impurity_frac</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>power</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="balanceFileLabel">
            <property name="text">
             <string>balance file</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="file_input"/>
            </item>
            <item>
             <widget class="QPushButton" name="file_read_button">
              <property name="text">
               <string>Read</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="designLabel">
            <property name="text">
             <string>design</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="designComboBox"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="sideLabel">
            <property name="text">
             <string>side</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="sideComboBox"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="spar_front_label">
            <property name="text">
             <string>SparFront</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QDoubleSpinBox" name="spar_front_input"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="runButton">
          <property name="text">
           <string>&amp;Run</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="textOutput">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="main_plot" native="true">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1691</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
